Src/PSharpRuntime/Values/PrtString.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
```

Comment:
This file should not be around? The folder PSharpRuntime is deleted.

Reply:
Yes, I'll delete this